internal/api/v2/types_gen.go

```diff
@@ -53,6 +53,11 @@ type QueryResponse struct {
 	Status      interface{}  `json:"status,omitempty" form:"status" query:"status" validate:"required"`
 }
 
+type DirectoryQueryResult struct {
```

Comment:
This was cloned from the v1 types_gen along with its MarshalBinary func because ExpandedEntries QueryResponse is a v2 type. The v1 version is still there because the v1 API is still using that.